Library/Homebrew/manpages/brew.1.md

```diff
@@ -444,6 +446,12 @@ With `--verbose` or `-v`, many commands print extra debugging information. Note
 
     Example: `brew install jruby && brew test jruby`
 
+  * `rm`, `remove`, `uninstall` [`--force`] <formula>:
+    Uninstall <formula>.
+
+    If `--force` is passed, and there are multiple versions of <formula>
+    installed, delete all installed versions.require "keg"
```

Comment:
I wonder where this trailing `require "keg"` came from!? (Re-running `brew man` eliminated it for me.)